src/openms/source/FORMAT/AbsoluteQuantitationMethodFile.cpp

```diff
@@ -38,176 +38,77 @@ namespace OpenMS
-    String feature_name = "";
-    if (headers.at("feature_name") != -1)
-    {
-      feature_name = line[headers.at("feature_name")];
-    }
-    aqm.setFeatureName(feature_name);
-    String IS_name = "";
-    if (headers.at("IS_name") != -1)
-    {
-      IS_name = line[headers.at("IS_name")];
-    }
-    aqm.setISName(IS_name);
-
-    // LODs
-    double llod = 0.0;
-    if (headers.at("llod") != -1)
-    {
-      llod = (line[headers.at("llod")].empty()) ? 0.0 : std::stod(line[headers.at("llod")]);
-    }
-    aqm.setLLOD(llod);
-    double ulod = 0.0;
-    if (headers.at("ulod") != -1)
-    {
-      ulod = (line[headers.at("ulod")].empty()) ? 0.0 : std::stod(line[headers.at("ulod")]);
-    }
-    aqm.setULOD(ulod);
-
-    // LOQs
-    double lloq = 0.0;
-    if (headers.at("lloq") != -1)
-    {
-      lloq = (line[headers.at("lloq")].empty()) ? 0.0 : std::stod(line[headers.at("lloq")]);
-    }
-    aqm.setLLOQ(lloq);
-    double uloq = 0.0;
-    if (headers.at("uloq") != -1)
-    {
-      uloq = (line[headers.at("uloq")].empty()) ? 0.0 : std::stod(line[headers.at("uloq")]);
-    }
-    aqm.setULOQ(uloq);
-
-    // concentration units
-    String concentration_units = "";
-    if (headers.at("concentration_units") != -1)
-    {
-      concentration_units = line[headers.at("concentration_units")];
-    }
-    aqm.setConcentrationUnits(concentration_units);
-
-    // statistics
-    Int n_points = 0;
-    if (headers.at("n_points") != -1)
-    {
-      n_points = (line[headers.at("n_points")].empty()) ? 0.0 : std::stoi(line[headers.at("n_points")]);
-    }
-    aqm.setNPoints(n_points);
-    double correlation_coefficient = 0.0;
-    if (headers.at("correlation_coefficient") != -1)
-    {
-      correlation_coefficient = (line[headers.at("correlation_coefficient")].empty()) ? 0.0 : std::stod(line[headers.at("correlation_coefficient")]);
-    }
-    aqm.setCorrelationCoefficient(correlation_coefficient);
-
-    // transformation model
-    String transformation_model = "";
-    if (headers.at("transformation_model") != -1)
-    {
-      transformation_model = line[headers.at("transformation_model")];
-    }
-    Param transformation_model_params;
-    for (auto const& kv : params_headers)
-    {
-      // cast doubles
-      std::vector<String> param_doubles {"slope", "intercept", "wavelength", "span", "delta", "x_datum_min", "y_datum_min", "x_datum_max", "y_datum_max"}; 
-      // cast integers
-      std::vector<String> param_ints {"num_nodes", "boundary_condition", "num_iterations"};
-
-      if (std::find(param_doubles.begin(), param_doubles.end(), kv.first) != param_doubles.end())
-      {
-        transformation_model_params.setValue(kv.first,std::stod(line[kv.second]));
-      }      
-      else if (std::find(param_ints.begin(), param_ints.end(), kv.first) != param_ints.end())
-      {
-        transformation_model_params.setValue(kv.first,std::stoi(line[kv.second]));
-      }
-      else
+    StringList tl = line; // trimmed line
```

Comment:
Good idea trimming out the white spaces

Reply:
It removes only whitespaces on the boundaries. The spaces within the name are not removed.